lib/crypted_hash.rb

```diff
@@ -1,2 +1,34 @@
+require "crypted_hash/cipher"
+
 class CryptedHash
+  class DecryptionError < Exception; end
+
+  attr_reader :secret, :hash
```

Comment:
Why `@secret` and `@hash` are publicly visible? It should not be in the API at all.